cmd/aspect/printaction/printaction.go

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 Aspect Build Systems, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package printaction
+
+import (
+	"github.com/spf13/cobra"
+
+	"aspect.build/cli/pkg/aspect/printaction"
+	"aspect.build/cli/pkg/aspect/root/flags"
+	"aspect.build/cli/pkg/interceptors"
+	"aspect.build/cli/pkg/ioutils"
+)
+
+func NewDefaultPrintActionCmd() *cobra.Command {
+	return NewPrintActionCmd(ioutils.DefaultStreams)
+}
+
+func NewPrintActionCmd(streams ioutils.Streams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "print_action",
+		Short: "Prints the command line args for compiling a file.",
+		Long: `Builds the specified targets and prints the extra actions for the given
```

Comment:
nit: can these multi-line literals be indented better like by escaping the first newline

```
        `\
First line goes here
more stuff`,
```

Reply:
they are all like this so could do a follow-up cleanup PR to fix them all up